samples/sensor/accel_polling/sample.yaml

```diff
@@ -0,0 +1,27 @@
+sample:
+  name: Accelerometer polling sample
+tests:
+  sample.sensor.accel_polling:
+    harness: console
+    tags: sensors
+    filter: dt_alias_exists("accel0")
+    platform_exclude: lpcxpresso55s69_cpu1
```

Comment:
why is this required?

Reply:
I found correct way to modify dts files about lpcxpresso55s69 board.
Remove this line.